File: inspect_geojson.py
import json
import os

file_path = os.path.join('public', 'data', '道路附属设施.geojson')

try:
    with open(file_path, 'r', encoding='utf-8') as f:
        data = json.load(f)
        if data['features']:
            print(json.dumps(data['features'][0]['properties'], ensure_ascii=False, indent=2))
except Exception as e:
    print(f"Error: {e}")

File: inspect_geojson.js
const fs = require('fs');
const path = require('path');

// 读取一个小文件进行测试
const filePath = path.join(__dirname, 'public', 'data', '道路附属设施.geojson');

try {
    const content = fs.readFileSync(filePath, 'utf8');
    const json = JSON.parse(content);
    if (json.features && json.features.length > 0) {
        console.log('Sample properties:', JSON.stringify(json.features[0].properties, null, 2));
    } else {
        console.log('No features found.');
    }
} catch (err) {
    console.error(err);
}
